### DISCUSSION TITLE
Google Season of Docs 2021 participation

### DISCUSSION THREAD

**Discussion by vfdev-5** [pytorch/ignite#1708] [Ideas]
We think to apply for a participation in Google Season of Docs 2021 with the [following proposal](https://github.com/pytorch/ignite/wiki/Google-Season-of-Docs-2021---Organization-Proposal).

Please, let us know here what do you think about that if you have any particular feedback.

We are also looking for volunteers to help us with few roles asked by Google Season of Docs. We encourage our community members to sign up to help with the following specific tasks:
- help us to recruit, evaluate, and onboard a technical writer
- be available to answer questions
- provide other support during the project (~4-5 months, including the application period)
- help us with coordination of the project: writing monthly evaluations, the final report and case study

If you would like to help us, please ping me here or on Discord (vfdev-5#4458) or send an email to contact@pytorch-ignite.ai
There are stipends expected for the volunteer work. Please, see "Project budget" section of the proposal.

GSoD Timelime: https://developers.google.com/season-of-docs/docs/timeline

TODO list:
- [x] Validate the proposal before March 16, 2021
- [x] Create a PR here : https://github.com/google/season-of-docs/blob/main/2021-participants/interested-organizations.md

**Comment by afzal442** [comment]
Hi there, I am interested in this proposal. @vfdev-5 could you look at your gitter account.

**Reply by vfdev-5** [reply]
Hi @afzal442 , thanks for your interest in this proposal! Can you detail your interest, thanks !
And, please use the indicated channels for the communications.

**Reply by afzal442** [reply]
Hi @vfdev-5, as per the requirement on the GSOD_GitHub repo, I have added myself onto gsod interested participant's list. I share the link here as [GSOD_INT](https://github.com/google/season-of-docs/blob/main/2021-participants/interested-technical-writers.md). You can find my details out there in the list and let me know if I have to reach out to you or any of the volunteers for any detail. Thanks.

**Reply by afzal442** [reply]
Looking forward to seeing next!

**Comment by vfdev-5** [comment]
🎉 We are excited to have our application for Google Season of Docs accepted : https://developers.google.com/season-of-docs/docs/participants/ 🎉

**Reply by sdesrozis** [reply]
Yeah a very good news !!

**Comment by vfdev-5** [comment]
Few details for tech writers interested in working on our proposal. Please, [send us](https://github.com/pytorch/ignite/wiki/Google-Season-of-Docs-2021---Organization-Proposal#contact-info) :
- your CV and links to your previous works or portfolio
- [project statement](https://developers.google.com/season-of-docs/docs/tech-writer-statement)

**Reply by vfdev-5** [reply]
We are ended our hiring process and will be selecting the best profile between all interviewed candidates by the end of this week. Please, do not consider above message : https://github.com/pytorch/ignite/discussions/1708#discussioncomment-629310 anymore.

**Comment by vfdev-5** [comment]
> We are excited to announce that Priyansi (@iiverveii) will work with us through Google Season of Docs 2021 ! We aim to audit and rewrite our documentation, simplify the learning curve for beginners and increase community engagement by incorporating user feedback in every step.

https://twitter.com/pytorch_ignite/status/1391751111443427330

We would like also to thank everyone who applied to our organisation and wrote the project statement !
